tests/common_tests/helpers/generate_test_tp_model.py

```diff
@@ -39,7 +39,7 @@ def generate_test_tp_model(edit_params_dict, name=""):
     base_config, op_cfg_list, default_config = get_op_quantization_configs()
 
     # separate weights attribute parameters from the requested param to edit
-    weights_params_names = [name for name in schema.AttributeQuantizationConfig.__init__.__code__.co_varnames if
+    weights_params_names = [name for name in schema.AttributeQuantizationConfig.model_fields.keys() if
```

Comment:
Can we maybe create a property within the class to return this? because `.model_fields.keys()` is kind of cryptic